x-pack/plugins/observability_solution/apm/public/components/routing/templates/service_group_template.tsx

```diff
@@ -137,6 +137,7 @@ export function ServiceGroupTemplate({
       environmentFilter={environmentFilter}
       showServiceGroupSaveButton={!isAllServices}
       showServiceGroupsNav={isAllServices}
+      showEnablementCallout={true}
```

Comment:
nit
```suggestion
      showEnablementCallout
```

Reply:
I don't know why I have the tendency to always write the boolean :P

Reply:
I always write it too. I thought there was a lint rule did this.